### ISSUE TITLE
Readme Issues

### ISSUE THREAD

**Issue by danverde** [byuitechops/katana#142]

**Comment by danverde** [comment]
Katana keeps complaining about a missing Firebase auth file. That needs to be explained in step 5

**Comment by danverde** [comment]
Also, your setup steps list ```npm start``` as the command to start the server, but it should really be ```npm run fullbuild```

**Comment by danverde** [comment]
Missing links at the bottom of the page

**Comment by danverde** [comment]
add compodoc as a dev dependency which needs to be installed globally

**Comment by sethchilders92** [comment]
Add step: Install @angular/cli
Step 5 -> (deploy firebase) needs to be broken up (and I think the link is incorrect)
-The document it links to breaks it up pretty clearly. The link now works if you on the github pages page

Step 7 -> How do I know what directory I want to deploy from? Do I actually have to do this?
- Typically Angular apps aren't deployed like we are doing apparently. If they have a server then they stick the server files on the server instead of integrating them into the Angular. Soooo...?

Step 8 -> I would mention URL + default port here
- It is mentioned under Development after step 9. Should it be moved here?

Katana keeps complaining about a missing Firebase auth file. That needs to be explained in step 5
- It is mentioned in the link in step 5, now

Also, your setup steps list npm start as the command to start the server, but it should really be npm run fullbuild
- It is mentioned after step 9 under Development

Missing links at the bottom of the page
- Still working on this

Are any of the above issues, besides missing links at the bottom of the page, not considered resolved now?

Thanks Daniel!

**Comment by danverde** [comment]
Add step: Install @angular/cli -> You're right we don't have to do this.
step 5 -> The link to Firebase setup in step 5 currently leads to a 404 page if you access it from the GitHub readme preview. It is a lot clearer though.
step 7 -> Currently it's not being used by anyone with a server, so I would write it in a way that makes sense for our office.
step 8 -> Anytime you introduce a new way to run the server I would give the default ports, The docs don't explain that `npm fullbuild` & `npm start` are the same command under the hood, so it would be good to give the port in both places. It might be a good idea to have Katana mention the port it's running on in the console output

**Comment by sethchilders92** [comment]
@danverde want to go over it again and see if there is anything else to change?

**Comment by danverde** [comment]
Do we need to include steps 1, 2, & 3? I would hope anyone who's gonna jump into creating their own mass maintenance tool in Node would know enough to install the repo...

**Comment by sethchilders92** [comment]
True, but I think it'd be fine to keep for when we do open it up to the world. It doesn't hurt to include it...?
